.eslintrc.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
     'IS_ESM': 'readonly',
     'IS_SXG': 'readonly',
     'IS_MINIFIED': 'readonly',
-    'IS_FORTESTING': 'readonly',
+    'IS_PROD': 'readonly',
```

Comment:
Historically, it was assumed that `minified` meant... well, minified code, and `fortesting` meant the output could be used to run tests (irrespective of whether the code was unminified or minified). However, we never shipped unminified code to production.

A couple questions (not necessarily an action item for this PR):

- How will `IS_MINIFIED` and `IS_PROD` differ after this change? Does that difference make sense?
- Does this PR create the opportunity to further simplify / consolidate their usage in source code?

Reply:
> However, we never ship unminified code to production, so prod kinda sorta implied minified code.

What do you mean `prod`? From what I can tell, there never was a concept called prod.

- As you call out, IS_PROD implies IS_MINIFIED. But the inverse is not true.
- There is room to simplify everything in the mode object.

Reply:
> What do you mean prod? From what I can tell, there never was a concept called prod.

Back in the day, the word "prod" was overloaded to mean two things: 1) stable channel AMP releases, and 2) the production config settings (see [here](https://github.com/ampproject/amphtml/tree/main/build-system/global-configs)). Today, we use "stable" for 1) but still use "prod" for 2).